modules/currency-l1/src/main/scala/org/tessellation/currency/l1/domain/dataApplication/consensus/Engine.scala

```diff
@@ -95,9 +97,9 @@ object Engine {
 
     implicit val dataEncoder = dataApplication.dataEncoder
 
-    def peersCount = 2
-    def timeout = 10.seconds
-    def maxDataUpdatesToDequeue = 50
+    def peersCount = appConfig.dataConsensus.peersCount.value
+    def timeout = appConfig.dataConsensus.timeout
+    def maxDataUpdatesToDequeue = appConfig.dataConsensus.maxDataUpdatesToDequeue.value
```

Comment:
It's okay to pass whole `appConfig` but it would be better to pass just `dataConsensus`, so here you'd have just `dataConsensusCfg.peersCount.value` and so on